pilot/pkg/networking/core/v1alpha3/route/retry/retry.go

```diff
@@ -29,8 +29,8 @@ import (
 // DefaultPolicy gets a copy of the default retry policy.
 func DefaultPolicy() *route.RetryPolicy {
 	policy := route.RetryPolicy{
-		NumRetries:           &types.UInt32Value{Value: 10},
-		RetryOn:              "connect-failure,refused-stream,unavailable,cancelled,resource-exhausted",
+		NumRetries:           &types.UInt32Value{Value: 2},
```

Comment:
HostSelectionMaxRetry attempts is set to 3.. seems inconsistent?

Reply:
I was only handling the 503s however I can also update with that as well

Reply:
This should now be set to 2